### ISSUE TITLE
Combo box updated color for a11y

### ISSUE THREAD

**Issue by colinreedmiller** [vmware/clarity#6577]
Change the border on the pill to meet 3.1 contrast.

Light theme: #737373
Dark theme: #8F9BA3

**Comment by gnomeontherun** [comment]
@Shijir does your color token work resolve this issue?

**Comment by colinreedmiller** [comment]
it certainly would be better if this was a core token rather than a hex value - I specified NG hex because it is a common color used in many places in NG - if we prefer a core token I can specify that instead.

**Comment by gnomeontherun** [comment]
@colinreedmiller for awareness, we have added a way to enable people to use the Core tokens with Clarity Angular, which should resolve most, if not all, contrast issues.

**Comment by colinreedmiller** [comment]
> @colinreedmiller for awareness, we have added a way to enable people to use the Core tokens with Clarity Angular, which should resolve most, if not all, contrast issues.

This is great and ideal provided NG contrast issues get fixed this way by products.

**Comment by Shijir** [comment]
I just checked the border's color contrast with the new shim installed. With the shim, the contrast improves but still fails to meet the 3:1 ratios.

The shim sets the pill's border color to `--cds-alias-object-border-color` which is `#cbd4d8`.
And the containing background (combobox's input background) is  `--cds-alias-object-container-background-shade` which is `#e3eaed`

Without Core tokens shim:
<img width="373" alt="Screen Shot 2022-01-18 at 4 37 22 PM" src="https://user-images.githubusercontent.com/3958008/150041471-44bf3160-872e-4b75-ac38-8eb293055b09.png">

With Core tokens shim:
<img width="371" alt="Screen Shot 2022-01-18 at 4 36 23 PM" src="https://user-images.githubusercontent.com/3958008/150041473-b6947d9c-6488-418a-abbb-542ab6c513e2.png">

**Comment by colinreedmiller** [comment]
These tokens will make the combobox pill meet contrast:
The containing background (combobox's input background):
`--cds-alias-object-container-background-tint`
and keep the pill's border color:
`--cds-alias-object-border-color`
<img width="1033" alt="Screen Shot 2022-01-18 at 5 47 24 PM" src="https://user-images.githubusercontent.com/44213553/150048235-0634962d-ddeb-4aa0-857b-42c782977aa3.png">

**Comment by Jinnie** [comment]
Done.

**Comment by Jinnie** [comment]
I had it assigned in the internal a11y tracking.

**Comment by Shijir** [comment]
> These tokens will make the combobox pill meet contrast: The containing background (combobox's input background): `--cds-alias-object-container-background-tint` and keep the pill's border color: `--cds-alias-object-border-color` <img alt="Screen Shot 2022-01-18 at 5 47 24 PM" width="1033" src="https://user-images.githubusercontent.com/44213553/150048235-0634962d-ddeb-4aa0-857b-42c782977aa3.png">

@colinreedmiller When I apply the token values you've suggested above, the pill border contrast looks a bit different than what's shown in your screenshot. In your screenshot, the pill border color looks a bit darker so I wonder whether that is `--cds-alias-object-border-color` or not.

<img width="468" alt="Screen Shot 2022-01-31 at 10 05 24 AM" src="https://user-images.githubusercontent.com/3958008/151848518-7a208361-e8e1-40cd-906a-7345f0f7166c.png">

**Comment by colinreedmiller** [comment]
Correct Core token for border is `--cds-alias-object-interaction-border-color`
